lib/rubocop/cop/layout/space_around_operators.rb

```diff
@@ -141,7 +141,7 @@ def offense(type, operator, with_space, right_operand)
         end
 
         def offense_message(type, operator, with_space, right_operand)
-          if operator.is?('**')
+          if operator.is?('**') && !space_around_exponet_operator?
```

Comment:
This changes offense the `CyclomaticComplexity` and `PerceivedComplexity` rule.
Should I consider refactoring this method in this PR?
![image](https://user-images.githubusercontent.com/4011729/70515557-7e736400-1b70-11ea-9500-0136a00a1b42.png)

Reply:
Yes, please. We spent a lot of energy bringing the complexity of RuboCop internals down. 🙏